docs/src/types/jump_types.md

```diff
@@ -116,15 +163,26 @@ and the internals will automatically build the `JumpSet`. `save_positions` is th
 `save_positions` argument built by the aggregation of the constant rate jumps.
 
 Note that a `JumpProblem`/`JumpSet` can only have 1 `RegularJump` (since a
-`RegularJump` itself describes multiple processes together).
-
-## Constant Rate Jump Aggregator
-
-The constant rate jump aggregator is the method by which the constant rate jumps
-are lumped together. This is required in all algorithms for both speed and accuracy.
-The current methods are:
+`RegularJump` itself describes multiple processes together). Similarly, it can
+only have one `MassActionJump` (since it also describes multiple processes
+together).
+
+## Constant Rate Jump Aggregators
+
+Constant rate jump aggregators are the methods by which constant rate
+jumps, including `MassActionJump`s, are lumped together. This is required in all
+algorithms for both speed and accuracy. The current methods are:
+
+- `Direct`: the Gillespie Direct method SSA.
+- `DirectFW`: the Gillespie Direct method SSA with `FunctionWrappers`. This
+  aggregator uses a different internal storage format for collections of
+  `ConstantRateJumps`. For systems with ~10 or more `ConstantRateJumps` it
+  should offer better performance than `Direct`.
+- `FRM`: the Gillespie first reaction method SSA. `Direct` should generally
+  offer better performance and be preferred to `FRM`.
+- `FRMFW`: the Gillespie first reaction method SSA with `FunctionWrappers`.
+  `DirectFW` should generally offer better performance and be preferred to `FRMFW`.
```

Comment:
It would probably be good to have a suggestion that recommends which aggregators, much like it's done for the ODE solvers.